tvb/simulator/monitors.py

```diff
@@ -514,6 +521,21 @@ def sample(self, step, state):
         if step % self._period_in_steps == 0:
             time = (step - self._period_in_steps / 2.0) * self.dt
             sample = self._state.copy() / self._period_in_steps
+
+            # By: Adam Li - Needs to Be Checked
+            # add observation noise if available
+            '''
+            1. Should dt be the same as in the source level?
```

Comment:
isn't this change done in the other PR? if so, handle my comments there on that branhc and then we can rebase this one on that, before merging this PR

Reply:
Yes, I think I made a mistake when I was modifying files on this branch. I reverted the file back to what is on the trunk of the master branch. 

I updated the other branch (obs-noise) with updates to this.